uavcan/equipment/ahrs/1002.PositionAttitude.uavcan

```diff
@@ -0,0 +1,40 @@
+#
+# Inertial data and orientation in body frame with fused location
+#
+
+uavcan.Timestamp timestamp
+
+float64 longitude                   # angular degree
+float64 latitude                    # angular degree
+float32 height_ellipsoid            # meter
+float32 height_msl                  # meter
+float32 height_baro                 # meter
+float32 ground_distance             # meter (with radar altimeter or LIDAR)
```

Comment:
Shall we rename `ground_distance` into `height_agl` for consistency?

Also, that's a lot of heights, but I don't have any better ideas, so it seems that we're going to have to live with this now.